pkg/cmd/openshift/openshift.go

```diff
@@ -15,10 +15,11 @@ import (
 	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
 	ktemplates "k8s.io/kubernetes/pkg/kubectl/util/templates"
 
+	cmdversion "github.com/openshift/oc/pkg/cli/version"
+	osversion "github.com/openshift/oc/pkg/version"
```

Comment:
That looks weird, one should be `openshift/origin/pkg/version` and other should be `openshift/oc/pkg/version`.

Reply:
clarification:

- `pkg/version` is where we set the internal vars during build time to commit/version/tag/etc..
- `pkg/cli/version` is the actual `oc version` command :-)